arcgis-ios-sdk-samples/Scenes/Get elevation at a point/README.md

```diff
@@ -14,10 +14,10 @@ Tap anywhere on the surface to get the elevation at that point. Elevation is rep
 
 ## How it works
 
-1. Create a scene view and scene with an imagery base map.
+1. Create an `AGSSceneView` and an `AGSScene` with an `imagery()` basemap.
 2. Set the elevation source of the scene's base surface.
 3. Convert the tapped screen point into a point on surface.
-4. Asynchronously get the elevation and display the results.
+4. Asynchronously get the elevation using the `AGSSurface.elevation(for:completion:)` method and display the results.
```

Comment:
Could you remove the redundant pound marks on line 29?
I was designing the rules for parsing README and found this file to be mal-formatted.

Reply:
GitHub does not support review on unchanged lines or refer to lines without sha hash, so I just raise it here.